plugins/flytekit-papermill/flytekitplugins/papermill/task.py

```diff
@@ -307,3 +311,35 @@ def record_outputs(**kwargs) -> str:
         lit = TypeEngine.to_literal(ctx, python_type=type(v), python_val=v, expected=expected)
         m[k] = lit
     return LiteralMap(literals=m).to_flyte_idl()
+
+
+def serialize_inputs(**kwargs) -> Dict[str, str]:
+    """
+    Serializes the inputs and saves separate files. Returns a dictionary
+    """
+    if kwargs is None:
+        return {}
+
+    outputs = {}
+    ctx = FlyteContext.current_context()
+    for k, v in kwargs.items():
+        expected = TypeEngine.to_literal_type(type(v))
+        lit = TypeEngine.to_literal(ctx, python_type=type(v), python_val=v, expected=expected)
+
+        tmp = tempfile.mktemp(suffix="bin")
+        utils.write_proto_to_file(lit.to_flyte_idl(), tmp)
+        outputs[k] = tmp
+
+    return outputs
+
+
+def read_input(path: str, dtype: T) -> T:
+
+    if type(path) == dtype:
```

Comment:
Sometimes users might fill in default values in the parameters cell in a papermill notebook. We still want the notebooks to work in this case.